recipes/drivelib/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "drivelib" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8e00c9981225ed7eae9f217f12242d98130aff59c2ff3e6026570589968a8fe3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # py<37
```

Comment:
```suggestion
```

Reply:
@marcelotrevisani when I tried to do this instead by requiring 'python >=3.7' I get a linter warning.  Should I still do it that way?

Reply:
You can add this package as ``noarch: python`` and let the python constrains as I suggested

Reply:
Please see here for more information
https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

Reply:
Here you can find one example of recipe which uses the same constrains and it is ``noarch: python``
https://github.com/conda-forge/grayskull-feedstock/blob/master/recipe/meta.yaml